skins/6ix.scss

```
@@ -0,0 +1,19 @@
+.six {
+    --primary-color: white;
+    --bg-color: #131313;
+    --text-color: white;
+    --link-color: white;
+    --heading-color: white;
+    --border-color: #eccece;
+    --header-bg: #131313;
+
+    #topnav .router-link-active,
+    #topnav .logo {
+        font-size: 0;
+        line-height: 0;
+        width: 0;
+        height: 0;
+        display: none;
+    }
+}
```

Please use only theme variables. No extra custom styles are allowed except for #topnav.
```suggestion
    #topnav {
        font-size: 0;
        line-height: 0;
        width: 0;
        height: 0;
        display: none;
    }
}
```